spec/components/shared/searchable_collection_component_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe Shared::SearchableCollectionComponent, type: :component do
+  let(:form) { instance_double(GOVUKDesignSystemFormBuilder::FormBuilder) }
+
+  let(:collection) do
+    [1, 2, 3, 4, 5].freeze
+  end
+
+  before do
+    allow(form).to receive(:govuk_collection_radio_buttons)
+    allow(form).to receive(:govuk_collection_check_boxes)
+  end
+
+  context "Radio button searchable collection" do
+    let(:radio_collection) do
+      described_class.new(form: form,
+                          attribute_name: :attributes,
+                          collection: collection,
+                          text_method: :first,
+                          hint_method: :first,
+                          value_method: :first,
+                          threshold: 10,
+                          scrollable: true).with_variant(:radiobutton)
+    end
+
+    describe "renders radio button variant" do
```

Comment:
As a convention we should use describe only for methods. In this case I would use 
`it "renders radio button variant" do`